### PR TITLE
doc: Update software maturity tables with nRF54L05 and nRF54L10

### PR DESCRIPTION
**Pull request by totyz**
Update Sidewalk software maturity tables for nRF54L05 and nRF54L10

### CODE REVIEW

**Review by divipillai — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Comment by LuDuda** [comment]
As discussed F2F, adding also updates for Matter and Thread.

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -837,6 +879,8 @@ The following table indicates the software maturity levels of the support for ea
         - Supported
         - Supported
         - Experimental
+        - -
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:882]:
```suggestion
        - --
```
This needs to be change for all the below cases, which has "-".

**Review by divipillai — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -851,6 +895,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:898]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -865,6 +911,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:914]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 4]
```diff
@@ -879,6 +927,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:930]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 5]
```diff
@@ -893,6 +943,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:946]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 6]
```diff
@@ -907,6 +959,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:962]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 7]
```diff
@@ -921,6 +975,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:978]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 8]
```diff
@@ -935,6 +991,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:994]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 9]
```diff
@@ -949,6 +1007,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1010]:
```suggestion
        - --
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 10]
```diff
@@ -963,6 +1023,8 @@ The following table indicates the software maturity levels of the support for ea
...
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:1026]:
```suggestion
        - --
```

**Review by divipillai — COMMENTED** [review]
Still missing for the following.

**Review by divipillai — APPROVED** [review]

**Review by mariusz-nordicsemi — APPROVED** [review]

**Pull request merged**